test/planners.jl

```diff
@@ -158,6 +158,15 @@ spec = SymbolicPlanners.BackwardSearchGoal(bw_spec, bw_state)
 
 end
 
+@testset "BiDirectional Planner" begin
+  planner = BiDirectionalPlanner(;heuristic=HAddR())
+  sol = planner(blocksworld, bw_state, bw_spec)
+  spec = SymbolicPlanners.BackwardSearchGoal(bw_spec, bw_state)
+  @test is_goal(spec, blocksworld, sol.trajectory[1])
+  @test collect(sol) == @pddl("(pick-up a)", "(stack a b)",
+                              "(pick-up c)", "(stack c a)")
```

Comment:
I suspect that one reason why this test is currently passing is because the BiDirectionalPlanner will eventually succeed as long as either the forward search succeeds or the backward search succeeds, even if the frontier crossing code isn't working quite right.

I think it would be good to design a more rigorous test case that actually checks if the frontier crossing code is working -- maybe this could be done by inspecting the search trees in the returned solution and seeing whether they cross?